canonical-websites/www.ubuntu.com
# Add Black to linting

Also renamed the CI endpoints to match many of the other projects.

If you're happy with this, when before you merge it, go into settings and change the required CircleCI tests to the newly named ones.

## QA

`./run` and check the site works and looks fine.

Check the new CircleCI tests pass.

If you're happy with this, when before you merge it, go into settings and change the required CircleCI tests to the newly named ones.
